package.json

```diff
@@ -37,8 +37,8 @@
 	},
 	"homepage": "https://github.com/dequelabs/axe-cli#readme",
 	"dependencies": {
-		"axe-core": "^3.0.0",
-		"axe-webdriverjs": "^2.0.0",
+		"axe-core": "^3.1.2",
+		"axe-webdriverjs": "^2.1.0",
```

Comment:
The npm repo says that the latest is 2.1.0, but the github repo only has a 2.0.0 release. Will that cause any problems?

Reply:
The GitHub repository not having a 2.1.0 tag is a problem. We should fix that.

No, this will not be a problem; dependencies are installed from npm (not GitHub).

Reply:
> The GitHub repository not having a 2.1.0 tag is a problem. We should fix that.

We should create an issue for this, so we do not forget to fix it.